src/data/data.ts

```diff
@@ -7,19 +9,27 @@ const formValue = function (label: string, multiplier: number): FormValue {
   return { label, multiplier }
 }
 
+const oneTimeMult = 0.06
 const housemateMult = 0.3
+const partnerMult = 0.48
 export const Interaction: { [key: string]: FormValue } = {
   oneTime: {
-    label: 'One-time interaction [6% chance of transmission per hour]',
-    multiplier: 0.06,
+    label: `One-time interaction [${fixedPointPrecisionPercent(
+      oneTimeMult,
+    )} chance of transmission per hour]`,
+    multiplier: oneTimeMult,
   },
   repeated: {
-    label: 'Household member [30% chance of transmission per week]',
+    label: `Household member [${fixedPointPrecisionPercent(
```

Comment:
Oooh, I don't know if it's a good idea to have the calculator use a different value from the label. I'm worried its going to confuse someone when the output results aren't actually doubling.

Reply:
I think either expose the 48% or change the value that the calculator uses to 50%.

Reply:
Oh I see. Well we already have this problem to some degree because the explanation text below has been showing 50% for a while: 
![image](https://user-images.githubusercontent.com/32290/95119740-4a6dbe80-0701-11eb-838c-6cae3c49b083.png)

Reply:
I'm happy to go either way. What's currently is in this PR is consistent with that explanation text. I think we could make the change you're suggesting in this PR or in a separate thread. I'll follow your lead.

Reply:
Yeah, I see that this is all over the place. I'll bring it up in a different bug.